CHANGELOG.rst

```diff
@@ -6,6 +6,7 @@ CHANGELOG
 ==========
 
 * doc-fix: Remove incorrect parameter for EI TFS Python README
+* enhancement: tox.ini: Add sphinx-build check for readthedocs
```

Comment:
Do we add build changes to changelog?

Reply:
I added for the twine one lol: https://github.com/aws/sagemaker-python-sdk/pull/651

Reply:
This one had an update: https://github.com/aws/sagemaker-python-sdk/pull/38